File: java/src/warranty/Claim.java
package warranty;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class Claim {

	public Claim(int id, double amount, Date date) {
		super();
		this.id = id;
		this.amount = amount;
		this.date = date;
	}
	public int id;
	public double amount;
	public Date date = new Date();
	public List<RepairPO> repairPO = new ArrayList<RepairPO>();
}

File: php74/src/Warranty/Claim.php
<?php

declare(strict_types=1);

namespace DDDCodeSamples\Warranty;

use DateTime;

/**
 * Class Claim
 *
 * @package Warranty
 */
class Claim
{
    public int $id;
    public float $amount;
    public DateTime $date;
    /** @var RepairPO[] */
    public array $repairPO;

    public function __construct(int $id, float $amount, DateTime $date)
    {
        $this->id = $id;
        $this->amount = $amount;
        $this->date = $date;
    }
}
